index.html

```diff
@@ -5024,30 +5021,43 @@ <h3 id="playback-AudioBufferSourceNode">
             The description of the algorithm is as follows:
           </p>
           <ol>
-            <li>Let <code>start</code> correspond to the <code>when</code>
-            parameter to <a href=
-            "#widl-AudioBufferSourceNode-start-void-double-when-double-offset-double-duration">
-              <code>start()</code></a> if supplied, else 0.
-            </li>
-            <li>Let <code>stop</code> correspond to the sum of the
-            <code>when</code> and <code>duration</code> parameters to
-              <a href="#widl-AudioBufferSourceNode-start-void-double-when-double-offset-double-duration">
-              <code>start()</code></a> if supplied, or the <code>when</code>
-              parameter to <code>stop()</code>, otherwise <em>Infinity</em>.
-            </li>
-            <li>Let <code>buffer</code> be the <a>AudioBuffer</a> employed by
-            the node.
-            </li>
-            <li>Let <code>offset</code> correspond to the <code>offset</code>
-            parameter to <a href=
-            "#widl-AudioBufferSourceNode-start-void-double-when-double-offset-double-duration">
-              <code>start()</code></a> if supplied, else 0. This is the
-              <a>playhead position</a> in the buffer at which playback will
-              begin.
-            </li>
-            <li>Let <code>loop</code>, <code>actualLoopStart</code> and <code>
-              actualLoopEnd</code> be the loop-related attributes of the node,
-              as discussed in the preceding section on looping.
+            <li>Initialize state for a new node:
+              <ol>
+                <li>Let <code>start</code> correspond to the <code>when</code>
+                parameter to <a href=
+                "#widl-AudioBufferSourceNode-start-void-double-when-double-offset-double-duration">
+                  <code>start()</code></a> if supplied, else 0.
+                </li>
+                <li>Let <code>stop</code> correspond to the sum of the
+                <code>when</code> and <code>duration</code> parameters to
+                <a href=
+                "#widl-AudioBufferSourceNode-start-void-double-when-double-offset-double-duration">
+                  <code>start()</code></a> if supplied, or the
+                  <code>when</code> parameter to <code>stop()</code>, otherwise
+                  <em>Infinity</em>.
```

Comment:
I think this is not quite defined since stop() can be called after the buffer has been started.  The algorithm as it stands now kind of implies that stop() is called before the rendering starts.

I think the processing loop needs to check for a call to stop() and update the stop value appropriately.